src/euler.jl

```diff
@@ -0,0 +1,97 @@
+# This file is a part of AstroLib.jl. License is MIT "Expat".
+
+function _euler{T<:AbstractFloat}(ai::T, bi::T, select::Integer, FK4::Bool, radian::Bool)
+
+    if select>6 || select<1
+        error("Input for coordinate transformation should be an integer in the range 1:6")
+    end
+
+    if FK4
+        psi = [0.57595865315, 4.9261918136, 0, 0, 0.11129056012, 4.7005372834]
```

Comment:
All these variables (`psi`, `stheta`, `ctheta`, `phi`) aren't really used as vectors, but as collections of elements that will be picked up later.  I'd suggest using tuples here, but replace `0` with `0.0`, so they'll be homogeneous tuples.  In this way memory allocations in `_euler` should drop to 0.